package.json

```diff
@@ -19,7 +19,11 @@
     }
   },
   "lint-staged": {
-    "*.{ts,tsx}": [
+    "!(*d).ts": [
+      "eslint --fix --max-warnings 0",
+      "git add"
+    ],
+    "*.tsx": [
```

Comment:
Should we add `.ts` here?

Reply:
**.ts** is catered for by the filter above it. I currently don't know how to combine them into one filter so I have separated them